collect_app/src/main/java/org/odk/collect/android/views/SlidingTabLayout.java

```diff
@@ -203,7 +204,14 @@ private void populateTabStrip() {
             }
 
             if (mTitleFontColor != -1) {
-                tabTitleView.setTextColor(getResources().getColor(mTitleFontColor));
+
+                int tabTitleTextColor;
```

Comment:
Did you look at the [`ContextCompat` class](https://developer.android.com/reference/android/support/v4/content/ContextCompat.html)? It has a method that does what you want.